spec/functions/packages_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe 'katello_devel::package' do
+  describe 'without scl' do
+    it { is_expected.to run.with_params().and_return() }
```

Comment:
I'd expect this to error out since it requires a non-empty string as first argument.